staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -153,6 +153,10 @@ type LeaderElectionConfig struct {
 	// simultaneously acting on the critical path.
 	ReleaseOnCancel bool
 
+	// DisableLockCreation should be set to true if the lock should not be created,
```

Comment:
nit: some clarifications: 

"the lock should not be created **when it doesn't exist**"
By default (when DisableLockCreation is false), leader elector creates a new lock object when trying to acquire a non-existing lock object.
If DisableLockCreation is true and the lock object doesn't exist, the leader elector won't be able to become the leader.